tests/Operation/CreateIndexesFunctionalTest.php

```diff
@@ -216,9 +213,7 @@ function (array $event): void {
 
     public function testCommitQuorumUnsupported(): void
     {
-        if (version_compare($this->getServerVersion(), '4.3.4', '>=')) {
-            $this->markTestSkipped('commitQuorum is supported');
-        }
+        $this->skipIfServerVersionNotSupported('>=', '4.3.4', 'commitQuorum is supported');
```

Comment:
Note: this is one example where "NotSupported" in the method name doesn't actually make sense. We're intentionally skipping a test where a feature _is_ supported in the server.

Likewise for some `hint` tests below.